handler/area_types.go

```diff
@@ -183,13 +198,30 @@ func (h *AreaTypes) GetParents(w http.ResponseWriter, r *http.Request) {
 	resp.Count = res.Count
 	resp.TotalCount = res.TotalCount
 	for _, e := range res.Dataset.Variables.Edges[0].Node.IsSourceOf.Edges {
+
+		i := 0
+		if len(e.Node.Meta.ONSVariable.GeographyHierarchyOrder) > 0 {
+			i, err = strconv.Atoi(e.Node.Meta.ONSVariable.GeographyHierarchyOrder)
+			if err != nil {
+				h.respond.Error(ctx, w, http.StatusBadRequest, &Error{
+					err: errors.Wrap(err, "unable to cast geography order field to int"),
```

Comment:
We probably want to return a more user friendly error message to the user (using the `msg`field` of the error

Reply:
Error field now updated